src/components/Whiteboard/ElementBehaviors/Resizable/ResizeHandle.tsx

```diff
@@ -17,101 +17,127 @@
 import { Dispatch, RefObject, useCallback, useRef } from 'react';
 import { DraggableCore, DraggableData, DraggableEvent } from 'react-draggable';
 import { useSvgCanvasContext } from '../../SvgCanvas';
-import { ResizeHandlePosition } from './types';
+import { LineElementProps, PolylineAndShapeElementsProps } from './types';
 
 export function calculateResizeHandlePosition(
-  position: ResizeHandlePosition,
-  containerWidth: number,
-  containerHeight: number,
-  scale: number,
+  params: { scale: number } & (
+    | LineElementProps
+    | PolylineAndShapeElementsProps
+  ),
```

Comment:
It could be better to split `params` of elements and `scale` to have two separate parameters here: `params: LineElementProps | PolylineAndShapeElementsProps` and `scale: number` that will simplify handling of `params` in the places where `calculateResizeHandlePosition` is used. Maybe some better name then `params` should be used.